typescript/api/services/DoiService.ts

```diff
@@ -157,17 +172,21 @@ export module Services {
           }
         }
 
-        let response = await instance.put(`/dois/${doi}`, putBody);
+        let response = await instance.put(`/dois/${doi}`, putBody)
         if (response.status == 200) {
-          return true;
+          return true
         } else {
-          sails.log.error("Unexpected response from DataCite API")
-          sails.log.error(response)
-        }
+          let errorMessage = TranslationService.t(response)
+          let customError: RBValidationError = new RBValidationError(errorMessage)
```

Comment:
Please create a doi minting unexpected error language code with a more generic message and send that here instead. We should log the actual error to the sails logs. These messages could appear to researchers as well as administrators and a json blob from the response is not user friendly (plus a potential security risk if too much is revealed in the response object)